src/goose/utils/check_shell_command.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+
+def is_dangerous_command(command: str) -> bool:
+    """
+    Check if the command matches any dangerous patterns.
+
+    Args:
+        command (str): The shell command to check.
+
+    Returns:
+        bool: True if the command is dangerous, False otherwise.
+    """
+    dangerous_patterns = [
+        r"\brm\b",  # rm command
```

Comment:
nit: I think `rm` on a single file might be fine, but I am scared of `rm -r` - especially combined with the file path checks below. Wdyt?

Reply:
better safe than sorry.

Reply:
A bit late, but I think it'd be nice to add some user flexibility here, possibly via config setting somewhere. eg: one might also not want to allow `curl`, `http`, etc.